domains/transfer/Transfer.API/API/Transfer/Api/v2024_01_03/Controllers/TransferAgreementProposalController.cs

```diff
@@ -164,10 +152,6 @@ public async Task<ActionResult> DeleteTransferAgreementProposal(Guid id)
         }
 
         var user = new UserDescriptor(User);
-        if (proposal.ReceiverCompanyTin != null && user.Organization!.Tin != proposal.ReceiverCompanyTin)
```

Comment:
I don't get this, what does this have to do with API versioning?

Reply:
Nothing.